app/services/authn_context_resolver.rb

```diff
@@ -78,6 +84,19 @@ def result_with_sp_aal_defaults(result)
     end
   end
 
+  def decorate_acr_result_with_user_context(result)
+    return result unless result.biometric_comparison?
+
+    return result if user&.identity_verified_with_biometric_comparison? ||
+      biometric_is_required?(result)
+
+    if user&.identity_verified?
+      result.with(biometric_comparison?: false, two_pieces_of_fair_evidence?: false)
+    else
+      result.with(biometric_comparison?: true)
+    end
```

Comment:
I'm a little behind on terminology -- is `two_pieces_of_fair_evidence` our current/legacy flow? if so it seems like we want to leave that out of this first branch?

Reply:
@jmhooper can you advise? my understanding was that `two_pieces_of_fair_evidence` is only for biometric comparison, so if biometric comparison is "preferred" but the user is already identity verified, we are falling back on that legacy flow, so it's not needed. apologies if that's incorrect, just let me know so i can update accordingly

Reply:
looking at the [vectors of trust implementation](https://github.com/18F/identity-idp/pull/10871), i think `two_pieces_of_fair_evidence is something biometric-comparison related, so i think this is right

Reply:
thanks for confirming!